tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -268,6 +269,64 @@ Object.Pipeline {
 			}
 		}
 	}
+
+	gain-capture.1 {
+		format		$FORMAT
+		index		16
+		Object.Widget.pipeline.1 {
+			stream_name	`"DMIC Raw 1"
+		}
+		Object.Widget.copier.1 {
+			stream_name	"DMIC Raw 1"
+			num_audio_formats 2
+			Object.Base.audio_format.1 {
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_bit_depth		32
+				out_valid_bit_depth	32
+				dma_buffer_size "$[$ibs * 2]"
+			}
+			Object.Base.audio_format.2 {
+				in_channels		4
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_channels		4
+				out_bit_depth		32
+				out_valid_bit_depth	32
+				dma_buffer_size "$[$ibs * 2]"
+				in_ch_cfg	$CHANNEL_CONFIG_3_POINT_1
```

Comment:
3.1 makes no sense here.

Reply:
@plbossart Will CHANNEL_CONFIG_QUATRO be better?

Reply:
probably yes.